core/domain/cron_services.py

```diff
@@ -28,7 +28,14 @@
 
 from mapreduce import model as mapreduce_model
 
-(job_models,) = models.Registry.import_models([models.NAMES.job])
+(base_models, job_models, user_models) = models.Registry.import_models([
```

Comment:
Do we need `job_models`?

Reply:
Yep, line 177.

Reply:
I see, thanks for sharing the line no. :)